homeassistant/components/zha/core/channels/__init__.py

```diff
@@ -1,403 +1,318 @@
-        report_config=(
-            REPORT_CONFIG_MIN_INT,
-            REPORT_CONFIG_MAX_INT,
-            REPORT_CONFIG_RPT_CHANGE,
-        ),
+        component: str,
+        entity: zha_typing.CALLABLE_T,
+        unique_id: str,
+        channels: List[zha_typing.ChannelType],
     ):
-        """Configure attribute reporting for a cluster.
-
-        This also swallows DeliveryError exceptions that are thrown when
-        devices are unreachable.
-        """
-        attr_name = self.cluster.attributes.get(attr, [attr])[0]
+        """Signal new entity addition."""
+        if self.zha_device.status == zha_core_device.DeviceStatus.INITIALIZED:
+            return
 
-        kwargs = {}
-        if self.cluster.cluster_id >= 0xFC00 and self.device.manufacturer_code:
-            kwargs["manufacturer"] = self.device.manufacturer_code
-
-        min_report_int, max_report_int, reportable_change = report_config
-        try:
-            res = await self.cluster.configure_reporting(
-                attr, min_report_int, max_report_int, reportable_change, **kwargs
-            )
-            self.debug(
-                "reporting '%s' attr on '%s' cluster: %d/%d/%d: Result: '%s'",
-                attr_name,
-                self.cluster.ep_attribute,
-                min_report_int,
-                max_report_int,
-                reportable_change,
-                res,
-            )
-        except (zigpy.exceptions.DeliveryError, Timeout) as ex:
-            self.debug(
-                "failed to set reporting for '%s' attr on '%s' cluster: %s",
-                attr_name,
-                self.cluster.ep_attribute,
-                str(ex),
-            )
-
-    async def async_configure(self):
-        """Set cluster binding and attribute reporting."""
-        if not self._zha_device.skip_configuration:
-            await self.bind()
-            if self.cluster.is_server:
-                for report_config in self._report_config:
-                    await self.configure_reporting(
-                        report_config["attr"], report_config["config"]
-                    )
-                    await asyncio.sleep(uniform(0.1, 0.5))
-            self.debug("finished channel configuration")
-        else:
-            self.debug("skipping channel configuration")
-        self._status = ChannelStatus.CONFIGURED
-
-    async def async_initialize(self, from_cache):
-        """Initialize channel."""
-        self.debug("initializing channel: from_cache: %s", from_cache)
-        self._status = ChannelStatus.INITIALIZED
-
-    @callback
-    def cluster_command(self, tsn, command_id, args):
-        """Handle commands received to this cluster."""
-        pass
-
-    @callback
-    def attribute_updated(self, attrid, value):
-        """Handle attribute updates on this cluster."""
-        pass
+        self.zha_device.hass.data[const.DATA_ZHA][component].append(
+            (entity, (unique_id, self.zha_device, channels))
```

Comment:
Please rename `entity` to entity_class`. That's what this is, right?

Reply:
Done.